packages/rocketchat-slashcommands-me/me.js

```diff
@@ -0,0 +1,20 @@
+
+/*
+ * Me is a named function that will replace /me commands
+ * @param {Object} message - The message object
+ */
+const Me = function(command, params, item) {
```

Comment:
Yes please, as that helps with stacktraces so that the functions in the traces are named and not "anonymous"